src/main.py

```diff
@@ -237,6 +241,51 @@ async def info(interaction: Interaction) -> None:
     await bc.send_stats(interaction)
 
 
+# Preview command
+@client.slash_command(dm_permission=False)
+async def preview(
+    interaction: Interaction,
+    uploaded_file: Attachment = SlashOption(description="The file to submit."),
+    submit_message: Optional[str] = SlashOption(
+        required=False, description="The submission message text."
+    ),
+) -> None:
+    """Show a preview of a submission's "Now Playing" embed."""
+    attachment_filepath = discord_utils.attachment_local_filepath(
+        interaction.id, uploaded_file
+    )
+    random_emoji = random.choice(config.emoji_list)
+    await uploaded_file.save(fp=attachment_filepath)
```

Comment:
I would wait until after we've determined the file is valid media before saving it to disk.

Reply:
Also, could you leave a comment explaining why we need to save the file as well, if we're only going to throw it away later? (I know why, it's just not obvious at a glance :)